paper-input-error.html

```diff
@@ -65,20 +65,5 @@
     behaviors: [
       Polymer.PaperInputAddonBehavior
     ],
-
-    properties: {
-      /**
-       * True if the error is showing.
-       */
-      invalid: {
```

Comment:
I mean it's a public property on an element that you can use. That's the definition of a breaking change